bindata/network/ovn-kubernetes/managed/alert-rules-control-plane.yaml

```diff
@@ -209,3 +209,111 @@ spec:
+      for: 15m
+      labels:
+        severity: warning
+    - alert: OVNKubernetesSouthboundDatabaseInboundConnectionMissing
+      annotations:
+        summary: OVN southbound database server(s) do not have expected number of inbound connections.
+        description: OVN southbound database server(s) do not have expected number of inbound connections which may
+          indicate degraded OVN database high availability.
+      expr: |
+        # Expected sum of inbound connections is number of control plane nodes * number of control plane nodes minus one
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        sum(max_over_time(ovn_db_cluster_inbound_connections_total{db_name="OVN_Southbound"}[5m]))
+        != count(max_over_time(kube_node_role{role="master"}[5m])) * (count(max_over_time(kube_node_role{role="master"}[5m]))-1)
+      for: 15m
+      labels:
+        severity: warning
+    - alert: OVNKubernetesNorthboundDatabaseOutboundConnectionMissing
+      annotations:
+        summary: OVN northbound database server(s) do not have expected number of outbound connections.
+        description: OVN northbound database server(s) do not have expected number of outbound connections which may
+          indicate degraded OVN database high availability.
+      expr: |
+        # Expected sum of outbound connections is number of control plane nodes * number of control plane nodes minus one
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        sum(max_over_time(ovn_db_cluster_outbound_connections_total{db_name="OVN_Northbound"}[5m]))
+        != count(max_over_time(kube_node_role{role="master"}[5m])) * (count(max_over_time(kube_node_role{role="master"}[5m]))-1)
+      for: 15m
+      labels:
+        severity: warning
+    - alert: OVNKubernetesSouthboundDatabaseOutboundConnectionMissing
+      annotations:
+        summary: OVN southbound database server(s) do not have expected number of outbound connections.
+        description: OVN southbound database server(s) do not have expected number of outbound connections which may
+          indicate degraded OVN database high availability.
+      expr: |
+        # Expected sum of outbound connections is number of control plane nodes * number of control plane nodes minus one
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        sum(max_over_time(ovn_db_cluster_outbound_connections_total{db_name="OVN_Southbound"}[5m]))
+        != count(max_over_time(kube_node_role{role="master"}[5m])) * (count(max_over_time(kube_node_role{role="master"}[5m]))-1)
```

Comment:
is this right? What are these outbound connections to? If it's SBDB connections on 9642 it would be to northd, as well as all ovn-controllers, and ovnkube-masters. If its raft it would just be 2 right?

Reply:
Its RAFT. 2 is correct. I count the number of leaders and subtract one.

Reply:
should the summary then indicate these are raft connections? 
OVN southbound database server(s) do not have expected number of outbound raft connections